### PR TITLE
added to jake's changes

### PR DESCRIPTION
**Pull request by satra**
some changes on top of your changes

### CODE REVIEW

File: dandi/models.py [hunk 1]
```diff
@@ -213,11 +216,18 @@ class ContactPoint(DandiBaseModel):
 
 
 class Contributor(DandiBaseModel):
-    identifier: Identifier = Field(None, nskey="schema")
+    identifier: Identifier = Field(
+        None,
+        title="A common identifier",
```
Comment (jjnesbitt) [inline dandi/models.py]:
```suggestion
        title="A Common Identifier",
```

File: dandi/models.py [hunk 2]
```diff
@@ -268,7 +278,9 @@ class Software(DandiBaseModel):
 class EthicsApproval(DandiBaseModel):
     """Information about ethics committee approval for project"""
 
-    identifier: Identifier = Field(nskey="schema")
+    identifier: Identifier = Field(
+        nskey="schema", title="Identifier string for approved protocol"
```
Comment (jjnesbitt) [inline dandi/models.py]:
```suggestion
        nskey="schema", title="Identifier String for Approved Protocol"
```

File: dandi/models.py [hunk 3]
```diff
@@ -537,6 +549,7 @@ class CommonModel(DandiBaseModel):
 
     # Linking to this dandiset or the larger thing
     access: List[AccessRequirements] = Field(
+        title="Access type",
```
Comment (jjnesbitt) [inline dandi/models.py]:
```suggestion
        title="Access Type",
```

File: dandi/models.py [hunk 4]
```diff
@@ -634,10 +647,14 @@ class AssetMeta(CommonModel):
...
     # TODO: https://github.com/NeurodataWithoutBorders/nwb-schema/issues/320
-    license: Optional[List[License]] = Field(None, nskey="schema")
+    license: Optional[List[LicenseType]] = Field(
+        None, title="License", description="License of item", nskey="schema"
+    )
 
     contentSize: str = Field(title="Content Size", nskey="schema")
-    encodingFormat: Union[str, AnyUrl] = Field(title="Encoding Format", nskey="schema")
+    encodingFormat: Union[str, AnyUrl] = Field(
+        title="File encoding format", nskey="schema"
```
Comment (jjnesbitt) [inline dandi/models.py]:
```suggestion
        title="File Encoding Format", nskey="schema"
```

File: dandi/models.py [hunk 5]
```diff
@@ -213,11 +216,18 @@ class ContactPoint(DandiBaseModel):
...
     name: str = Field(None, nskey="schema")
     email: EmailStr = Field(None, nskey="schema")
     url: AnyUrl = Field(None, nskey="schema")
-    roleName: List[RoleType] = Field(title="Role Name", nskey="schema")
+    roleName: List[RoleType] = Field(
+        title="Role", description="Role of the contributor", nskey="schema"
+    )
     includeInCitation: bool = Field(
         True,
         title="Include contributor in citation",
```
Comment (jjnesbitt) [inline dandi/models.py]:
```suggestion
        title="Include Contributor in Citation",
```

**Comment by jjnesbitt** [comment]
@satra I just fixed capitalization, are these changes okay?

**Comment by satra** [comment]
yup - this is fine with me.

**Pull request merged**